### PR TITLE
add YAFFA

### PR DESCRIPTION
**Pull request by kantorge**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [ ] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/yaffa.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: YAFFA
+website_url: https://www.yaffa.cc
+source_code_url: https://github.com/kantorge/yaffa
+description: YAFFA is a personal finance web application, and can be used to keep track of your money, expenses, budgets, investments. It also helps with long term financial planning.
```
Comment (Rabenherz112) [inline software/yaffa.yml:4]:
```suggestion
description: Personal finance web application, that can be used to keep track of your money, expenses, budgets, and investments. It also helps with long-term financial planning.
```

**Review by Ki-er — APPROVED** [review]

**Review by kokomo123 — APPROVED** [review]
LGTM, no issues here pending other thoughts from collaborators. Thanks for your contribution!

**Review by nodiscc — CHANGES_REQUESTED** [review]
Installation instructions are extremely lacking https://www.yaffa.cc/documentation/resources/category/installation/

**Comment by kantorge** [comment]
> Installation instructions are extremely lacking https://www.yaffa.cc/documentation/resources/category/installation/

Thank you for your review and feedback. Indeed, there were a few draft pages, which are now all completed, and new, installation-related guides are also added. I checked a few examples from the already listed repositories, trying to meet the same level of detail. Please let me know if you recommend any further changes.

**Review by nodiscc — APPROVED** [review]

**Pull request merged**